Pike/github-pages-deploy-action
# Add DRY_RUN option, passing --dry-run to git push.

**Description**
Add a `DRY_RUN` option. See #499 for the proposal.

**Testing Instructions**
Unit tests included, to check at least one of the push commands for the `--dry-run` option.
